lib/palindrome_check.rb

```diff
@@ -1,5 +1,55 @@
 # A method to check if the input string is a palindrome.
 # Return true if the string is a palindrome. Return false otherwise.
+
+require 'pry'
+
+#with array and string
+# def palindrome_check(my_phrase)
```

Comment:
The algorithm will work well. However, it uses an additional string which increases the space complexity of your solution.